src/parse_args.ts

```diff
@@ -104,6 +111,20 @@ program
     '--quiet-exit-code',
     'always return 0 as an exit code status, regardless of test pass / fail. Only return > 0 exit codes on internal errors where the suite could not be run'
   )
+  .addOption(
+    new Option('--download-throughput <throughput>', 'Maximal aggregated download throughput (bytes/sec). -1 disables download throttling.')
+      .default(defaultNetworkConditions.downloadThroughput)
+  )
+  .addOption(
+    new Option('--upload-throughput <throughput>', 'Maximal aggregated upload throughput (bytes/sec). -1 disables upload throttling.')
+      .default(defaultNetworkConditions.uploadThroughput)
+  )
+  .addOption(
+    new Option('--latency <latency>', 'Minimum latency from request sent to response headers received (ms).')
+      .default(defaultNetworkConditions.latency)
+  )
+  .option('--offline', 'True to emulate internet disconnection.', defaultNetworkConditions.offline)
+  .option('--no-throttling', 'Turns off default throttling.')
```

Comment:
except for this option `no-throttling`, I would remove every other arg from CLI and only allow them to be configured only via synthetics config file if users needs customization.

Reply:
@vigneshshanmugam What about configuring via heartbeat, especially for inline scripts. Do we want to make this feature available for inline scripts?

Reply:
We need a way to solve this going forward with synthetics config file from HB for inline suites as we cannot keep adding CLI for each individual feature needing to be exposed via heartbeat flag. I would remove this from here and add a feature to HB to invoke them via a temp synthetics config file. 

@andrewvc Thoughts?

Reply:
It's an interesting question, I was actually thinking we could keep adding flags. I don't see why not? What's the downside? We still don't have nearly as many flags as, say, [cURL](https://linux.die.net/man/1/curl). 

One nice property of flags is that they break if the version of synthetics doesn't support them. This means that if a user intends to override a default that only works if the version of synthetics supports it. If it does not it fails loudly. 

A config file would probably fail silently, which IMHO is less desirable.

Reply:
There is nothing wrong in adding extra flags, But this nifty feature which is related to a small change results in 5 flags is not a desirable thing to do IMO. We should stick with a strict number of flags and allow fine tuning via config files as they allow more customization. 

> One nice property of flags is that they break if the version of synthetics doesn't support them.

We can do the same with config files. Its just a implementation detail as of now.

Reply:
One sort of middle path would be to combine the throttling options into a mini syntax like: `--throttle="5d/3u/2l"`.

I think you have a good point @v re: too many options being a bit unwieldy UX-wise in terms of having to wade through them, so this may strike a good balance.

Re: the idea of using a file for options, one really important property of CLI args is that replicating the commands heartbeat invokes is as simple as copy/pasting the heartbeat log line which shows the full command invoked by heartbeat. Having to replicate the config file too would make debugging tricky in support situations.

Reply:
As another note, we could parse the throttle config in any order, just split on `/` and look at the trailing letter to see what it's affecting.

Reply:
@vigneshshanmugam I know we discussed parsing it based on order, but I think making it order agnostic makes sense for cases in which you want to override one value while leaving other defaults. Otherwise if order mattered we'd have to have some token for skipping a value or the users would have to know what the defaults are and always provide every value.

Reply:
++ to order agnostic, it's more clear and not much more work

Reply:
++ Sounds good to me.